### DISCUSSION TITLE
Fixes only until October 1st

### DISCUSSION THREAD

**Discussion by worksofliam** [codefori/vscode-ibmi#321] [General]
Starting from tomorrow, the 24th of September, the Code for IBM i repository will only be accepting cleanup PRs or fix PRs.

To make sure we fix any existing issues that crop up, I am initiating a period of time where **only fixes of cleanups will be merged and released**. This is only until October 1st, and then we will go back to normal.

This doesn’t mean you shouldn’t work on PRs. But note that **I will not review or merge any PR that is for an enhancement** until October 1st. I will review and release fixes as normal.

During this time, I will be working on a couple of large feature issues and reviewing/fixing anything else that comes up.

Thanks,
Liam Barry

**Comment by szsascha** [comment]
Nice to see how active this project is getting :)
Feeling it's growing from week to week now.

**Reply by worksofliam** [reply]
Agree - and thank you for all your contributions!

**Reply by szsascha** [reply]
@worksofliam 

FYI
Will start working again from 1st Oct. on my current PR. Maybe I need a bit of support from your side. Don't want to distract you.

**Comment by priceaj** [comment]
Are there any issues which need to be worked on? I can't see anything marked as Bug at present.

**Reply by worksofliam** [reply]
No pressing issues right now. Over the next week I plan to make sure any new issues get answered, and then also need to plan the 0.8.0 release. Perhaps, if you'd like, you could list which enhancements you think would be worth me spending my time on.

Thanks for all your help in new issues and PRs.
